src/components/BaseCard.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="flex flex-col rounded-lg shadow-md overflow-hidden">
+    <div class="flex-shrink-0">
+      <img class="h-48 w-full object-cover" :src="post.attributes.src || 'https://i.stack.imgur.com/y9DpT.jpg'">
+    </div>
+    <div class="flex-1 bg-white p-6 flex flex-col justify-between">
+      <div class="flex-1">
+        <p class="text-sm font-medium text-indigo-600">
+          <a
```

Comment:
not sure why this is a link, where are not seding the user anywhere from this tag, better make it a span